### PR TITLE
Azure CI: Move openvino-lin to Ubuntu 20.04

### PR DESCRIPTION
**Pull request by azhogov**

### CODE REVIEW

File: install_build_dependencies.sh [hunk 1]
```diff
@@ -32,6 +32,7 @@ if [ -f /etc/lsb-release ]; then
     sudo -E apt update
     sudo -E apt-get install -y \
             build-essential \
+            cmake \
```
Comment (ilya-lavrenov) [inline install_build_dependencies.sh:35]: why? in this file below we download cmake 3.13 which is minimum required for OpenVINO build

File: install_build_dependencies.sh [hunk 2]
```diff
@@ -47,7 +48,11 @@ if [ -f /etc/lsb-release ]; then
             libtool \
             autoconf \
             shellcheck \
-            python \
+            patchelf \
+            libenchant1c2a \
+            python3-pip \
```
Comment (ilya-lavrenov) [inline install_build_dependencies.sh:53]: should not we use https://pip.pypa.io/en/stable/installation/#get-pip-py approach?

Comment (azhogov) [reply install_build_dependencies.sh:53]: Why do you need to download and compile cmake when OS cmake is suitable???

File: install_build_dependencies.sh (hunk 1 above)
Comment (azhogov) [reply install_build_dependencies.sh:35]: Why do you need to download and compile cmake when OS cmake is suitable???

File: install_build_dependencies.sh (hunk 2 above)
Comment (azhogov) [reply install_build_dependencies.sh:53]: patchelf libenchant1c2a is not python
pip is needed for installing using pip :)))

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**